### DISCUSSION TITLE
Flux Ui flux:editor component issue

### DISCUSSION THREAD

**Discussion by adanyal46** [livewire/flux#851] [Feature Requests]
When i use flux:editor component used in laravel so it show me error
Using $this when not in object context
![image](https://github.com/user-attachments/assets/c8dde92c-e4b4-4450-b017-c56ecd420bca)

**Comment by ju5t** [comment]
You can only use the editor within the context of a Livewire component.

**Reply by adanyal46** [reply]
coluld you explain with code?

**Reply by joshhanley** [reply]
@adanyal46 the editor component must be used inside a Livewire component. It seems like you may be using it in your layout or in a standard Laravel controller view, but that won't work.
